### PR TITLE
Colors and update to mdi icons

### PR DESCRIPTION
**Pull request by imostovenko**

### CODE REVIEW

File: components/layout/app-bar/app-bar.scss [hunk 1]
```diff
@@ -1,24 +1,24 @@
 .icon {
-  height: 44px;
-  width: 44px;
-  color: white;
+  height: 42px;
+  width: 42px;
+  color: white !important;
```
Comment (acharlop) [inline components/layout/app-bar/app-bar.scss]: also can't the color be set on the icon? `!important` is always a last resort

**Comment by acharlop** [comment]
I removed the font awesome stuff 💪

File: components/layout/app-bar/app-bar.scss [hunk 2]
```diff
@@ -1,24 +1,24 @@
 .icon {
-  height: 44px;
-  width: 44px;
-  color: white;
+  height: 42px;
```
[the remark this answers is not in the document]
Comment (imostovenko) [reply components/layout/app-bar/app-bar.scss]: i removed it from css. it still works OK.

File: components/layout/app-bar/app-bar.scss (hunk 1 above)
Comment (imostovenko) [reply components/layout/app-bar/app-bar.scss]: it could. done.

**Pull request merged**